src/components/interaction-designer/ResourceViewer.vue

```diff
@@ -23,10 +23,21 @@ export const DEBOUNCE_FLOW_PERSIST_MS = 1500
 export default {
   name: 'ResourceViewer',
   mixins: [Lang],
+  props: {
+    filteredBlocks: {
+      type: Array,
+      default: () => null,
```

Comment:
is `default` needed?

Reply:
I believe it's either required or some default value, null seemed reasonable for an array

Reply:
Sorry, I see now that we have an ESLint rule requiring a default value for everything that's not marked as required.

Reply:
The value of `() => null` is an interesting choice.
Have you considered using `null/undefined`?
I think the visibleBlocks could then use nullish coalescing like
`return this.filteredBlocks ?? this.activeFlow?.blocks ?? []`.